server/src/building/TargetBuildDatabase.cpp

```diff
@@ -0,0 +1,81 @@
+#include "TargetBuildDatabase.h"
+
+#include "loguru.h"
+#include "utils/GrpcUtils.h"
+#include "utils/GenerationUtils.h"
+
+TargetBuildDatabase::TargetBuildDatabase(BuildDatabase *baseBuildDatabase,
+                                         const fs::path &_target) :
+        BuildDatabase(baseBuildDatabase),
+        target(_target),
+        isAutoTarget(_target == GrpcUtils::UTBOT_AUTO_TARGET_PATH) {
```

Comment:
`isAutoTarget` is an easily computable property and doesn't need a backing field